src/lib/stable/globals.ts

```diff
@@ -38,6 +40,8 @@ declare global {
     // eslint-disable-next-line no-var
     var _azlePostUpgradeCalled: boolean;
     // eslint-disable-next-line no-var
+    var _azleRecordBenchmarks: boolean;
```

Comment:
Should we initialize this? Because if not our type isn't really correct...

Reply:
I was following the example, it looked like most of the other simple values where not being initialized. I was only seeing initialization for objects... I'll add it in for this one

Reply:
done